eahub/base/forms.py

```diff
@@ -19,3 +19,13 @@ def clean(self):
             ]  # Will raise a error message
         else:
             return cleaned_data["reasons"]
+
+
+class SendMessageForm(forms.Form):
+    your_name = forms.CharField(label="Name", max_length=10000, widget=forms.TextInput)
```

Comment:
Can we prepopulate this with their user name? This makes the experience smoother for the user and also makes clear *whose* name we're asking for here. Alternatively, change "name" to "Your name"